dependencies.yaml

```diff
@@ -118,7 +118,7 @@ dependencies:
           - dask-cuda=23.02.*
           - dask-cudf=23.02.*
           - ucx>=1.13.0
-          - ucx-py=0.29.*
+          - ucx-py=0.30.*
```

Comment:
Shouldn't ucx-py be getting pulled in transitively through raft?

Reply:
Correct, I just created this PR becuase I thought we would be able to merge it before #5038 but it seems that that PR is now ready, so will close this one when that one is merged